src/cameras/OrthographicCamera.js

```diff
@@ -15,10 +15,10 @@ function OrthographicCamera( left, right, top, bottom, near, far ) {
 	this.zoom = 1;
 	this.view = null;
 
-	this.left = left;
-	this.right = right;
-	this.top = top;
-	this.bottom = bottom;
+	this.left = ( left !== undefined ) ? left : -1;
```

Comment:
You have to write it like so, otherwise Travis fails:
```js
this.left = ( left !== undefined ) ? left : - 1;
```
Notice the space after the minus.

Reply:
Thank you for pointing out my mistake! :-) Consistent style is good (although I often prefer to skip spaces after signs)